app/views/categories/new.html.erb

```diff
@@ -1,9 +1,14 @@
-<h1>New category</h1>
-
+<h1 class="text-center">New category</h1>
+<div class="text-center"
+>
 <%= render "form", category: @category %>
 
+</div>
+
+
 <br>
 
-<div>
-  <%= link_to "Back to categories", categories_path %>
+<div class="text-center ms-4">
+  <%= link_to "Back to categories", categories_path, class:"btn btn-primary"
+%>
 </div>
```

Comment:
- [ ] Kindly improve the design of your category form to match the design for forms.

- [ ] Kindly ensure you add the back button ("<") as we have in the design and it should lead to the homepage when clicked.

  Your Implementation | Behance Design
  :---:|:---:
  ![irie](https://user-images.githubusercontent.com/54848363/214601859-dce635f2-f5e8-4dcc-90a1-aac11c42e0f5.jpg) | ![siow](https://user-images.githubusercontent.com/54848363/214601866-5d1d8527-bc7d-4036-a8bb-88db7d2ffc44.jpg)